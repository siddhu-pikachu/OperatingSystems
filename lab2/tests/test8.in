# ./(executable_name) -M test8.in

4 4
1,2,3,4,
5,6,7,8,
9,10,11,12,
1,8,5,9,

4 3
1,2,8,
2,3,4,
3,1,3,
4,9,6,
