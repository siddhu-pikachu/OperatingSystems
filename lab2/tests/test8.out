30,47,49,
70,107,133,
110,167,217,
68,112,109,
